autowebcompat/utils.py

```diff
@@ -80,6 +80,10 @@ def prepare_images():
 images = {}
 
 
+def load_pair(fname):
```

Comment:
Since you're introducing this function, make use of it in train.py and pretrain.py too.